app/state.py

```diff
@@ -0,0 +1,19 @@
+"""This module manages state for the application."""
+
+from typing import Dict
+
+from app.providers.provider_aws import ProviderAWS
+
+
+class ENPState:
+    """Custom application state class."""
+
+    def __init__(self) -> None:
+        """Initialize ENPState with a default set of providers."""
+        # Route handlers should access this dictionary to send notifications using
+        # various third-party services, such as AWS, Twilio, etc.
+        self.providers: Dict[str, ProviderAWS] = {'aws': ProviderAWS()}
```

Comment:
```suggestion
        self.providers: Dict[str, ProviderBase] = {'aws': ProviderAWS()}
```
Needs to be imported.

Reply:
Good catch, Kyle.

Reply:
Updated!